docs/design_docs/base-image.md

```diff
@@ -23,5 +23,5 @@ This image is based on a minimal debian image (currently `k8s.gcr.io/debian-base
 due to high availability of tooling.  
```

Comment:
nit: let's do:
docs/design
docs/user
docs/devel

less typing, conveys roughly the same information, especially considering we have titles and links everywhere

Reply:
:heavy_check_mark: